README.md

```diff
@@ -1,24 +1,212 @@
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
 
-* Ruby version
+<!-- PROJECT DESCRIPTION -->
 
-* System dependencies
+# 📖 [booking-app ] <a name="about-project"></a>
 
-* Configuration
+> In this application we have built a rails backend for a user model, doctors model ,and a reservations model to support a front end client for an application which allows users to book appointments with doctors
 
-* Database creation
+**[booking-app]** is an application which allows users to book appointments with doctors.
 
-* Database initialization
+## 🛠 Built With <a name="built-with"></a>
 
-* How to run the test suite
+### Tech Stack <a name="tech-stack"></a>
 
-* Services (job queues, cache servers, search engines, etc.)
+> This application was built using Ruby on Rails
 
-* Deployment instructions
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
 
-* ...
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[create_user]**
+- **[create_doctors]**
+- **[create_appointment]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://yourdeployedapplicationlink.com)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- Have ruby installed
+- Have node installed
+- Have rails installed
+
+### Setup
+
+You can find the React Frontend client at [React_client](https://github.com/j-tee/front-end-booking-app/tree/development)
+
+Clone this repository to your desired folder:
+
+To run and view this application, you will first have to clone a local version of this repository using:
+
+`git clone https://github.com/j-tee/booking-app.git`
+
+After that navigate to the newly created folder:
+
+`cd booking-app`
+
+### Install
+
+Install this project with:
+ `bundle install`
+
+### Usage
+
+To run the project, execute the following command:
+
+You can find the React Frontend client at [React_client](https://github.com/RayhanTabase/hello-react-front-end)
+
+Configure PostgreSQL database in your project in the project directory:
+
+`config/database.yml`
+
+Start a local server running the command:
+
+`rails s`
```

Comment:
- [ ] Kindly, please include in your instructions the db commands that your users will need to set up the database: `rails db:create, rails db:migrate, rails db:seed`.

That way, your users will be able to run your protect smoothly. 👍